backend/src/controllers/projects/contacts.js

```diff
@@ -3,4 +3,31 @@ const model = require("@models/projects/contacts");
 const what = { single: "project contact", plural: "project contacts" };
 const controller = useController(model, what, "projects");
 
+/**
+ * Update an item by ID. Use passed info from the request body.
+ *
+ * @param   {FastifyRequest} request FastifyRequest is an instance of the standard http or http2 request objects.
+ * @param   {FastifyReply}   reply   FastifyReply is an instance of the standard http or http2 reply types.
+ * @returns {object}
+ */
+controller.updateContacts = async (request, reply) => {
+  let contacts = [];
+  for (const [key, value] of Object.entries(request.body)) {
+    value.map((row) => {
+      contacts.push({
+        contact_role: Number(key),
+        project_id: Number(request.params.id),
+        contact_id: row.value,
```

Comment:
we should probably cast this to Number as well,

Reply:
To be safe yes